rfc/rfc008-certificate-structure.md

```diff
@@ -33,98 +33,47 @@ When it comes to certificates, we’d like to be at the edge of usability and se
+TSPs are responsible for signing certificates. The TSPs have their own CA. 
+To trust all PKIo certificates, any software that validates a certificate and its chain, MUST trust any intermediate CA below the **Staat der Nederlanden Private Services CA – G1** CA.
 
-Typically, a PKI infrastructure also defines CRLs and OCSP endpoints. More recent insights show that short-lived certificates offer the same level of security since CRLs and OCSP return values tend to be cached for a longer period of time. A case could even be made that short-lived certificates are more secure since they allow for faster key rotation as well. [Short lived certificates @ Netflix](https://medium.facilelogin.com/short-lived-certificates-netflix-fd5f3ae5bc9)
+The PKIoverheid private services CA is not by default accepted by browsers and operating systems. 
 
-The node certificates from the diagram are specific per machine/deployment. This allows for multiple topologies. The private key SHOULD never leave the deployment instance. Together with short-lived certificates, the private key can even be stored in memory rather than somewhere else, increasing security.
+### 3.2 Certification Practise Statement
+The different Certification Practise Statements for the Dutch government can be found on https://cps.pkioverheid.nl/
+The relevant CPS is the **CPS PA PKIoverheid Private Root v1.5**.
 
-The node signing certificates are used for signing documents \([RFC004 Distributed Document Format](https://github.com/nuts-foundation/nuts-specification/tree/4127385cab773917fd5bd642dd7f2fa94f48655e/rfc/rfc004-distributed-document-format.md)\) and for signing JWTs\([RFC003 OAuth2 authorization](rfc003-oauth2-authorization.md)\). The node TLS certificates are used for connections between nodes. They also are short-lived and are deployment instance specific.
+### 3.3 CRL
+The Certificate Revocation Lists for PKIoverheid certificates are maintained by the different TSPs. 
+The time between signing of a CRL differs per TSP. A node MUST at least update all CRLs every hour.
 
-Summarized, there are 3 classes of certificates: 1. Vendor CA certificates are used to issue certificates. 2. TLS certificates are used for authentication when connecting to node endpoints. 3. Signing certificates are used for signing documents on the document log, so their authenticity can be established. They are also used to sign JWTs in an OAuth flow.
+## 4. Test and development
+PKIoverheid certificates are not cheap and the associated private key should not be shared amongst developers or test servers.
```

Comment:
not cheap = expensive = relative

Better would be somewhere along the lines of "Might be relatively expensive for small organizations"